### ISSUE TITLE
False positive in navigation markdown.

### ISSUE THREAD

**Issue by stark1tty** [sindresorhus/awesome-lint#164]
Navigation markdown is squawking in lint. Could I please have some clarifications on setting parameters for exceptions? I realise there is documentation here, but I am finding it unclear (still learning, self teaching, sorry!). Do we need a separate document to list these or do they go in the text? If they go in the text, wont the html formatting show up?

Link to repo: https://github.com/stark1tty/Awesome-Historic_al-Maps

clip of output:

```
 readme.md:6:2
  ×    1:1    Missing Awesome badge after the main heading                                            remark-lint:awesome-badge
  ×    1:1    License was not detected by GitHub                                                      remark-lint:awesome-github
  ×    6:2    Add 1 space between block quote and content                                             remark-lint:blockquote-indentation
  ×   10:1    Table of Contents must be the first section                                             remark-lint:awesome-toc
  ×   12:1    ToC item "Databases and Search Engines" link "#Databases-and-Search-Engines" not found  remark-lint:awesome-toc
  ×   60:5    #Awesome-Historic_al-Maps                                                               remark-lint:double-link
  ×   76:5    #Awesome-Historic_al-Maps                                                               remark-lint:double-link
  ×   84:3    https://uwm.edu/lib-collections/agsl-digital-map-collection/                            remark-lint:double-link
  ×   86:69   List item description must end with proper punctuation                                  remark-lint:awesome-list-item
  ×   87:114  List item link and description must be separated with a dash                            remark-lint:awesome-list-item
  ×   88:57   List item link and description must be separated with a dash                            remark-lint:awesome-list-item
  ×   89:75   List item link and description must be separated with a dash                            remark-lint:awesome-list-item
  ×   90:75   List item link and description must be separated with a dash                            remark-lint:awesome-list-item
  ×   91:58   List item link and description must be separated with a dash                            remark-lint:awesome-list-item
  ×   96:171  List item description must end with proper punctuation                                  remark-lint:awesome-list-item
  ×   97:59   List item link and description must be separated with a dash                            remark-lint:awesome-list-item
  ×  109:5    #Awesome-Historic_al-Maps                                                               remark-lint:double-link
  ×  115:5    #Awesome-Historic_al-Maps                                                               remark-lint:double-link
  ×  128:3    https://www.narvik-bunkers.no/overview/                                                 remark-lint:double-link
```

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome-lint?tab=readme-ov-file#special-comments
